x-pack/plugins/uptime/server/lib/alerts/__tests__/status_check.test.ts

```diff
@@ -44,16 +45,21 @@ const bootstrapDependencies = (customRequests?: any) => {
  */
 const mockOptions = (
   params = { numTimes: 5, locations: [], timerange: { from: 'now-15m', to: 'now' } },
-  services = { callCluster: 'mockESFunction', savedObjectsClient: mockSavedObjectsClient },
+  services = alertsMock.createAlertServices(),
   state = {}
-): any => ({
-  params,
-  services,
-  state,
-});
-
-const mockSavedObjectsClient = { get: jest.fn() };
-mockSavedObjectsClient.get.mockReturnValue(defaultDynamicSettings);
+): any => {
+  services.savedObjectsClient.get.mockResolvedValue({
+    id: '',
+    type: '',
+    references: [],
+    attributes: defaultDynamicSettings,
+  });
```

Comment:
Hi @elastic/uptime 
Looks like the typing has unearthed a possible mistake in the Unit Test where the mock returned the dynamic settings *as* the saved object, instead of the *attributes* of the saved object. I think this fix is in line with the implementation.
Note that this change has updated the snapshots below, so please verify this is still correct. :)